libs/metall/src/dice/metall/kernel/segment_storage/umap_sparse_segment_storage.hpp

```diff
@@ -112,10 +112,10 @@ class umap_sparse_segment_storage {
     // TODO: implement parallel copy version
 
     if (clone) {
-      METALL_TRACE("Clone: {}", source_path);
+      METALL_TRACE("Clone: {} to {}", source_pathm, destination_path);
```

Comment:
more generally: we would need to have at least one unit test that is built with umap. otherwise, we will not really know if this code works.

Reply:
Will create an issue